config/environments/production.rb

```diff
@@ -4,6 +4,7 @@
   # Code is not reloaded between requests.
   config.paperclip_defaults = {
       :storage => :s3,
+      :s3_protocol => 'http',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.